storage/account.go

```diff
@@ -18,6 +19,11 @@ import (
 	"github.com/Azure/go-autorest/autorest/to"
 )
 
+func getAccountName() string {
+	accountName := "azuresamplesgo" + helpers.GetRandomLetterSequence(10)
+	return strings.ToLower(accountName)
```

Comment:
i think there is a length limit for storage account name, are we respecting it ?

Reply:
3 - 24 characters :)